server/migrations/env.py

```diff
@@ -82,6 +82,7 @@ def process_revision_directives(context, revision, directives):
         context.configure(
             connection=connection,
             target_metadata=target_metadata,
+            compare_type=True,
```

Comment:
Added this so that flask-migrate will detect column type changes.

https://stackoverflow.com/questions/33944436/flask-migrate-and-changing-column-type/52181159#52181159